charts/gardener-extension-provider-azure/templates/configmap-logging.yaml

```
@@ -16,7 +16,7 @@ data:
 
     [FILTER]
         Name                parser
-        Match               kubernetes.*machine-controller-manager*azure-machine-controller-manager*
+        Match               kubernetes.*machine-controller-manager*machine-controller-manager-provider-azure*
```

```suggestion
        Match               kubernetes.*azure-machine-controller-manager*machine-controller-manager-provider-azure*
```
Shouldn't this match with [this](https://github.com/gardener/gardener-extension-provider-azure/pull/257/files#diff-e38adde4c814c39f0c3443b395a76dfc43639b003e8514c731b39e2b1683aea9R71)?

---

I am not sure. But [this comment](https://github.com/gardener/gardener-extension-provider-azure/pull/257#discussion_r607122158) from Vladimir suggested to extend it to ConfigMap-logging. Did I comprehend it wrongly by any chance?

---

Your understanding is right. However both the container names must be matching right. Else we may end up not collecting logs from the other/old container.

---

Got it. Got it! Thanks @prashanth26